content/guides/user-guides/delfin/driver-matrix-support/huawei-ocenstor/_index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Huawei OceanStor
+menuTitle: Huawei OceanStor
+description: "Huawei OceanStor Support Matrix"
+weight: 20
+disableToc: false
+tags: ["user guide", "delfin", "drivers", "huawei"]
+---
+
+### Model and Versions Supported
+
+|Models   | Firmware Versions   |
```

Comment:
Please populate the content for the Huawei OceanStor driver support.

Reply:
This file should contain the values for Huawei Oceanstor driver.

Reply:
#144 has been created to add data